test/unit/generator/content/part2/get-reductions.test.js

```diff
@@ -0,0 +1,36 @@
+const getReductions = require('../../../../../app/generator/content/part2/get-reductions')
+
+describe('get reductions', () => {
+  test('returns empty string in array if no reductions', () => {
+    const result = getReductions([])
+    expect(result.length).toBe(1)
+    expect(result[0]).toBe('')
+  })
+
+  test('includes reductions header if reductions', () => {
+    const reductions = [{ id: 1, reason: 'Late claim submission' }]
```

Comment:
can we use `mockReductions` here and throughout?

const reductions = mockReductions[0]